src/compiler/types.ts

```diff
@@ -3968,6 +3968,7 @@ namespace ts {
         typeAcquisition: TypeAcquisition;               // Used to customize the type acquisition process
         compilerOptions: CompilerOptions;               // Used as a source for typing inference
         unresolvedImports: ReadonlyArray<string>;       // List of unresolved module ids from imports
+        containsCommonJsRequire: boolean;               // True if there is a `require()` call somewhere in the program

```

Comment:
Should this be made optional? This interface is casted to in one place but I can't find anywhere we actually create it -- but it does happen to share a shape with `interface DiscoverTypings` in `server.ts` so maybe it comes from there?